content/zh/docs/v3.4/cluster-administration/cluster-wide-alerting-and-notification/alerting-message.md

```diff
@@ -7,21 +7,62 @@ linkTitle: "告警消息（节点级别）"
 weight: 8540
 ---
 
-告警消息会记录按照告警规则触发的告警的详细信息。本教程演示如何查看节点级别的告警消息。
+KubeSphere 为节点提供告警规则，提供编组，允许用户将相似的规则编入一个规则组中，一旦满足这些规则定义的条件，将会触发告警。本教程将演示如何为集群中的节点创建规则组及告警规则。
+
+KubeSphere 还具有内置规则组。 在<strong>内置规则组</strong>选项卡，您可以点击任一规则组查看该规则组中所有规则，点击任一规则查看其详情。请注意，这些规则不能直接在控制台上进行删除，但可以通过编辑调整规则。
+
 ## 准备工作
 
-- 您需要启用 [KubeSphere 告警系统](../../../pluggable-components/alerting/)。
-- 您需要创建一个用户 (`cluster-admin`) 并授予其 `clusters-admin` 角色。有关更多信息，请参见[创建企业空间、项目、用户和角色](../../../quick-start/create-workspace-and-project/#step-4-create-a-role)。
-- 您已经创建节点级别的告警策略并已触发该告警。有关更多信息，请参考[告警策略（节点级别）](../alerting-policy/)。
+- 您需要启用 [KubeSphere 告警系统](../../pluggable-components/alerting)。
+- 如需接收告警通知，您需要预先配置[通知渠道](../platform-settings/notification-management/configure-email)。
+- 您需要创建一个用户 (cluster-admin) 并授予其 clusters-admin 角色。有关更多信息，请参见[创建企业空间、项目、用户和角色](../../quick-start/create-workspace-and-project#step-4-create-a-role)。
+-您需要确保集群中存在工作负载。如果尚未就绪，请参见[部署并访问 Bookinfo](../../quick-start/deploy-bookinfo-to-k8s) 创建一个示例应用。
```

Comment:
This line may be removed because this doc is for cluster level alerting, so the workload is not necessary